### DISCUSSION TITLE
DAI 2023-2024 - Practical work 3 - @AlexandrePhilibert and @valentin-ricard

### DISCUSSION THREAD

**Discussion by AlexandrePhilibert** [heig-vd-dai-course/heig-vd-dai-course#175] [Show and tell]
Link to the repository: https://github.com/AlexandrePhilibert/dai-labo-3

**Comment by knownasred** [comment]
Link to the last commit: [Last Commit](https://github.com/AlexandrePhilibert/dai-labo-3/commit/ce7b4d08eb9be1fbfec2298f637d975483b62ed2)
